README.md

```diff
@@ -424,10 +429,10 @@ but uses the resulting schema as the subject for validation.
 > Note! The `value` here is the _root_ value relative to the starting schema, not the value at the nested path.
 
 ```js
-const schema = object({
+var schema = object({
   foo: array().of(
     object({
-      loose: bool(),
+      loose: boolean(),
```

Comment:
I generally perfer bool since it's not a reserved word

Reply:
But `boolean` is not reserved word too, like `string`, `number` or others. Am i missing something?

In api section `boolean` is on the same levels as others: having section named `boolean` and listed in exports, mentioning only in comments that there is also `bool` alias.